aten/src/ATen/native/cpu/AtomicAddFloat.h

```diff
@@ -0,0 +1,28 @@
+#ifndef ATOMIC_ADD_FLOAT
+#define ATOMIC_ADD_FLOAT
+
+#include <immintrin.h>
+#include <atomic>
+
+static inline void cpu_atomic_add_float(float* dst, float fvalue)
+{
+  typedef union {
+    unsigned intV;
+    float floatV;
+  } uf32_t;
+
+  uf32_t new_value, old_value;
+  std::atomic<unsigned>* dst_intV = (std::atomic<unsigned>*)(dst);
```

Comment:
Can you please elaborate, why you are using `unsigned` here, instead of just using `float` and void union?

Reply:
This is need to compare with 4-byte value, the sizeof void is 1 ? for unsigned is 4-byte